packages/web/src/components/ui/Footer/Footer.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import styles from './Footer.module.scss';
+import Container from '../Container/Container';
+
+export default function Footer() {
+  return (
+    <Container>
+      <footer className={styles.footer}>
+        <div className={styles.copyright}>© 2022 GradeJS. All rights reserved.</div>
+        <div className={styles.wrapper}>
+          <div className={styles.links}>
+            <a
+              href='https://github.com/gradejs/gradejs/discussions/6'
+              target='_blank'
+              rel='norefferer noreferrer'
+              className={styles.link}
+            >
+              About
+            </a>
+            <a
+              href='https://github.com/gradejs/gradejs/discussions'
+              target='_blank'
+              rel='norefferer noreferrer'
```

Comment:
What is `noreFFerer`? Was it intended as `noopener noreferrer`?

Reply:
Didn't notice. Copied from current production Header links https://github.com/gradejs/gradejs/blob/develop/packages/web/src/components/ui/Header/Header.tsx#L13

Reply:
Nowadays probably should be just `<a href="..." rel="noreferrer" target="_blank">`. Ref: [https://html.spec.whatwg.org/multipage/links.html#link-type-noreferrer](https://html.spec.whatwg.org/multipage/links.html#link-type-noreferrer)